siriuspy/siriuspy/currinfo/main.py

```diff
@@ -2,8 +2,14 @@
 
 import time as _time
 from datetime import datetime as _datetime
+import logging as _log
+from copy import deepcopy as _dcopy
+
 import numpy as _np
 from epics import PV as _PV
+import visa
+
+from mathphys.functions import get_namedtuple
```

Comment:
use of  `_get_namedtuple` to keep module namespace API clean.

Reply:
done!